keys/keys.go

```diff
@@ -121,9 +121,15 @@ func RangeStatsKey(rangeID roachpb.RangeID) roachpb.Key {
 
 // SequenceCacheKey returns a range-local key by Range ID for a
 // sequence cache entry, with detail specified by encoding the
-// supplied client command ID.
-func SequenceCacheKey(rangeID roachpb.RangeID, family []byte) roachpb.Key {
-	return MakeRangeIDKey(rangeID, LocalSequenceCacheSuffix, encoding.EncodeBytes(nil, family))
+// supplied transaction ID and sequence number.
+// The length of the prefix of the encoded key excluding the encoded
+// sequence number is returned.
+func SequenceCacheKey(rangeID roachpb.RangeID, id []byte, seq uint32) (roachpb.Key, int) {
```

Comment:
Now that I look at it, it seems more awkward in this case because I'll wind up constructing another key when all I need is the length. I'm going to leave as is but let me know if I'm missing something.